aegis-web/yo/app/scripts/directives/extmdHeader.js

```diff
@@ -0,0 +1,14 @@
+ 'use strict';
+
+ angular.module('hopsWorksApp').directive('extmdHeader', function() {
```

Comment:
Can we name this more general? So we can extend it later to the header directive everybody uses.

Reply:
Yes, as discussed. Either we merge this now as initial version for the header and improve on that and change the name in further PRs or we can remove it from this PR.

Reply:
I think it's ok but we need a general one too that will include in the template both gray and white header parts of the current template and transclude the white part. breadcrumb should be part of it and/or be configurable via attributes.